test/stream.5.test.js

```diff
@@ -103,31 +104,31 @@ test('reply.send handles aborted requests', t => {
   })
 
   fastify.listen({ port: 0 }, err => {
-    t.error(err)
-    t.teardown(() => { fastify.close() })
+    t.assert.ifError(err)
+    t.after(() => fastify.close())
 
     const port = fastify.server.address().port
     const http = require('node:http')
     const req = http.get(`http://localhost:${port}`)
       .on('error', (err) => {
-        t.equal(err.code, 'ECONNRESET')
-        fastify.close()
+        t.assert.strictEqual(err.code, 'ECONNRESET')
+        done()
       })
 
     setTimeout(() => {
-      req.abort()
+      req.destroy()
```

Comment:
why this change?

Reply:
> why this change?

because is deprecated ( see screenshot )

<img width="1119" alt="method-abort-deprecated" src="https://github.com/user-attachments/assets/d4e02eb3-cf7a-439e-a7f5-ec4ceb576318" />

Reply:
@Eomm as soon as you have time, will you let me know if I need to make other changes or can it be joined ?